aeutils_eqc/aeu_mp_trees_eqc.erl

```diff
@@ -71,7 +74,7 @@ gen_ops(NOps) ->
 gen_ops(0, _Keys, Ops) -> return(lists:reverse(Ops));
 gen_ops(N, Keys, Ops) ->
     frequency(
-        [{2, ?LET({K, V}, {gen_put_key(Keys), gen_value()}, gen_ops(N-1, lists:umerge(Keys, [K]), [{put, K, V} | Ops]))},
+        [{2, ?LET({K, V}, {gen_put_key(Keys), gen_non_empty_value()}, gen_ops(N-1, lists:umerge(Keys, [K]), [{put, K, V} | Ops]))},
```

Comment:
it would have been sufficient to just write `non_empty(gen_value())` in this line and forget about the other change.

Reply:
It is a bit worrying that originally tests seem to have past. Something must have been changed after the model was completed... but it seems right to put non-empty values here, otherwise it would mean a delete.

Reply:
Little experiment: followed in the model the semantics that putting an empty value is removing it:

```
gen_ops(N, Keys, Ops) ->
    frequency(
        [{2, ?LET({K, V}, {gen_put_key(Keys), (gen_value())}, gen_ops(N-1, lists:umerge(Keys, [K]), [{put, K, V} || V =/= <<>>] ++
                                                                          [{delete, K} || V == <<>> ] ++ Ops))},
         {1, ?LET(K, gen_delete_key(Keys), gen_ops(N-1, Keys -- [K], [{delete, K} | Ops]))}]).
```
passing all tests as well... but then, non_empty is more elegant.